.github/workflows/rw-doc-storybook-deploy.yml

```diff
@@ -0,0 +1,52 @@
+# (C) 2024 GoodData Corporation
+
+name: rw ~ Doc ~ Storybook deploy
+on:
+  workflow_call:
+    inputs:
+      source-branch:
+        required: true
+        description: "The name of the source branch"
+        type: string
+        default: "master"
```

Comment:
required: true and default: master, should we make it optional or remove the default if it's required?

Reply:
Fixed